Doc/library/unittest.mock.rst

```diff
@@ -2095,6 +2095,10 @@ mock_open
    .. versionchanged:: 3.5
       *read_data* is now reset on each call to the *mock*.
 
+   .. versionchanged:: 3.8
```

Comment:
@ned-deily should I change 3.8 to 3.7.1 or remove the versionchanged directive? (I will submit a separate PR for master if we decide to remove it.)

Reply:
I think you should change it to 3.7.1 for the 3.7 branch. Thanks!